debezium-core/src/main/java/io/debezium/transforms/spi/QualifiedTableNameResolver.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms.spi;
+
+import io.debezium.converters.spi.RecordParser;
+
+/**
+ * A {@link java.util.ServiceLoader} interface that connectors should implement to resolve fully qualified table name.
+ *
+ * @author Mario Fiore Vitale
+ */
+public interface QualifiedTableNameResolver extends RecordParserProvider {
```

Comment:
Why do we need two separate interfaces?

Also please use name `QualifiedDataCollectionNameResolver` to make it datsource independent.

Reply:
This interface is to explicitly separate the concern of the two interface. One is for parsing the record and the other one define the resolve of a full data collection name.

Reply:
Then should one inherit from another? The class can use multpile interfaces.